src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/SpaceBookingSummaryStatusTestHelper.kt

```diff
@@ -0,0 +1,181 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.util
+
+import org.junit.jupiter.params.provider.Arguments
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1SpaceBookingSummaryStatus
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.unit.transformer.cas1.TestCaseForSpaceBookingSummaryStatus
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.stream.Stream
+
+object SpaceBookingSummaryStatusTestHelper {
+  private var testCaseIndex = 0
+
+  fun spaceBookingSummaryStatusCases(): Stream<Arguments> {
```

Comment:
- These test cases were first built (using TDD) to create the unit tests to drive the implementation. 
- The test cases are extensive and test the date boundaries fully (I hope!)
- It was moved here to be shared by the unit tests and integration tests
- I was mindful that using this for the integration tests might slow down the build too much as it creates 89 test cases in total
- I was also mindful that we discussed up front that we want the integration tests to be solid encase we move the business logic (to generate the statuses) to the sql query at some point (i.e. if we introduce filtering / sorting on status at some stage - though out of scope officially for now!)
- In the end, I decided to reuse this for both unit tests and integration tests and see how the build behaves. Here are the results:

1. The integration tests in the build before mine took `1m 55s`: 
<img width="1728" alt="before" src="https://github.com/user-attachments/assets/29b16978-5039-4356-827d-f5971ccfa3c8">

2. The integration tests in my build (with the extra 89 tests took `2m 09s`: 
<img width="1728" alt="after" src="https://github.com/user-attachments/assets/8d2deb09-5aa8-4f93-8203-d27c0199172f">

**Conclusion**: adding 14 secs to the build is not the end of the world and proves it works properly. 

*Happy to be contradicted on this conclusion if this is of concern and can slim down the integration tests to be 1 test case per status